src/debug-message/js/index.ts

```diff
@@ -85,12 +86,12 @@ export class JSDebugMessage extends DebugMessage {
           ? `${funcThatEncloseTheVar} ${delemiterInsideMessage} `
           : ''
         : ''
-    }${selectedVar}${quote}, ${selectedVar})${semicolon}`;
+    }${selectedVar}${logMessagePostfix}${quote}, ${selectedVar})${semicolon}`;
     if (wrapLogMessage) {
       // 16 represents the length of console.log("");
       const wrappingMsg = `console.${logType}(${quote}${logMessagePrefix} ${'-'.repeat(
         debuggingMsg.length - 16,
-      )}${logMessagePrefix}${quote})${semicolon}`;
+      )}${logMessagePostfix}${quote})${semicolon}`;
```

Comment:
I changed this part since there was no Postfix in the past and Prefix was being used at the end of wrappingMsg. I think it makes more sense to use the postfix if this get's added to the extension. Although we could revert it if it's interpreted as unwanted.